src/components/navbar/help-dropdown.vue

```diff
@@ -23,7 +23,9 @@ except according to the terms contained in the LICENSE file.
         <a href="https://forum.getodk.org/" target="_blank">{{ $t('common.forum') }}</a>
       </li>
       <li>
-        <a href="/version.txt" target="_blank">{{ $t('common.version') }}</a>
+        <a href="/version.txt" target="_blank" @click="showVersion">
```

Comment:
I was imagining this as a sort of fallback. If `centralVersion.dataExists`, then `event.preventDefault()` will be called, and you'll see the modal. But if something strange has happened such that `centralVersion` is never set, this will still provide an easy way to see /version.txt. My hope is to make sure that version information is available in some form for troubleshooting purposes.

An example of a situation in which this might come up is if the request for /client-config.json fails. In that case, we render a special error page. The user will still see Version in the navbar dropdown. However, we won't be able to show them the modal in that case, since `centralVersion` would not be set. Yet we can still link to /version.txt for them.